internal/commands/user/inputs.go

```diff
@@ -0,0 +1,110 @@
+package user
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+const (
+	userDataEmail = "email"
+	userDataName  = "name"
+)
+
+func validAuthProviderTypes() []interface{} {
+	apts := make([]interface{}, 0, len(realm.ValidAuthProviderTypes))
+	for _, apt := range realm.ValidAuthProviderTypes {
+		apts = append(apts, apt)
+	}
+	return apts
+}
+
+func displayUser(apt realm.AuthProviderType, user realm.User) string {
+	sep := " - "
+	display := apt.Display()
+	displayUserData, err := displayUserData(apt, user)
+	if err != nil {
+		return ""
+	}
+	if displayUserData != "" {
+		display += sep + displayUserData
+	}
+	return display + sep + user.ID
+}
+
+func displayUserData(apt realm.AuthProviderType, user realm.User) (string, error) {
```

Comment:
especially as this is just for display, I'd rather be a little resilient and just omit/display empty here